src/tree_annotation/core.cljs

```diff
@@ -304,9 +240,7 @@ This is an open source project. Find the code [here](https://github.com/DCMLab/t
 (set! (.-onkeydown js/document)
       (fn [event]
         (case (.-code event)
-          "Enter" (combine-nodes)
-          "Escape" (deselect-all-nodes)
-        (when (.-ctrlKey event)
-          (case (.-code event)
-            "KeyR" (start-rename-node)
-            "KeyD" (db/del-selected-nodes))))))
+          "Enter" (db/combine-selected!)
+          "Escape" (db/deselect-all!)
+          "Backspace" (db/delete-selected!)
+          "Delete" (db/delete-selected!))))
```

Comment:
The idea is that renaming is not related to selection. I don't think it makes sense to start renaming all selected nodes. Instead, it should be tied to a double click/tap on a single node. Since you need to use the mouse anyway to select nodes, a key for renaming doesn't save any time.

What would be the use case for renaming several nodes at the same time? Changing all labels of a certain kind together?